app/views/posts/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="my-bio bg-secondary bg-gradient">
+    <div class="container">
+        <div class="row">
+            <%= render partial: 'user', locals: { user: @user } %>
+        </div>
+        <% @posts.each do |post| %> 
+            <div class="card my-4">
+                <div class="card-header">
+                    <h4># <%= link_to post.Title, user_post_path(@user, post) %></h4>
+                </div>
+                <div class="card-body"><p class="mt-2"><%= truncate(post.text, length: 200, omission: '...') %></p></div>
+                <div class="media-info">
+                    <ul>
+                        <li><a href="#">Comments: <%=post.comments_count.present? ? post.comments_count : 0%></a></li>
+                        <li><a href="#">Likes: <%=post.likes_count.present? ? post.comments_count : 0%></a></li>
+                    </ul>
+                </div>
+                <div class="accordion">
+                    <div class="accordion-item">
+                        <h2 class="accordion-header">
+                            <button class="accordion-button" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                Comments
+                            </button>
+                        </h2>
+                        <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
+                            <div class="card-body">
+                                <!-- Single comment-->
+                                <%= link_to "Add a comment", new_user_post_comment_path(post_id: post.id, user_id: @user.id), class: "btn btn-primary"%>
```

Comment:
- [ ] Please fix this error that occurs with the user with posts.
![image](https://github.com/Izebiafe/Blog/assets/91486791/31c31d40-990d-45ca-907d-489410faaa29)